docs/examples/tutorial/profiling_tutorial/calc_pi_3.py

```diff
@@ -0,0 +1,16 @@
+# cython: profile=True
+
+@cython.cfunc
+@cython.inline
+@cython.exceptval(-1.0)
+def recip_square(i: int) -> cython.double:
```

Comment:
```suggestion
def recip_square(i: cython.int) -> float:
```
we can use `float` here i think it will look better
but `int` must be with `cython`

Reply:
To be honest, I prefer to have explicit `cython.double` here. Cython is automatically using `double` C type when variable is annotated by python `float` type. This can be confusing for the newcomers. I agree that it would look better but in tutorial I prefer to not introduce this hidden complexity.

Reply:
It has already been used in docs in new pythonized versions, so you are not introducing anything here.

But look at it like this: you are seeing cython for the first time and you don’t know that behind the scenes python `float` is actually `double`, and this is where you get even more confused, for pythonista it is easier to use the already known `float`.
And there is no hiding here, it is nicely described [here](https://cython.readthedocs.io/en/latest/src/tutorial/pure.html#pep-484-type-annotations)